taglib/tag.h

```diff
@@ -97,6 +97,12 @@ namespace TagLib {
     virtual String album() const = 0;
 
     /*!
+     * Returns the album artist name; if no album artist name is present in the
+     * tag String::null will be returned.
+     */
+    virtual String albumArtist() const = 0;
```

Comment:
As @scotchi metioned, we can't add virtual functions due to ABI compatibility.

To merge this pull request to our `master` branch, you need to change the function like this: https://github.com/taglib/taglib/blob/master/taglib/audioproperties.cpp#L108-L154
I know it's really ugly, but it's necessary to emulate a virtual function without breaking the ABI.